client/api/README.md

```diff
@@ -137,13 +137,13 @@ func convert_internal_ComputeDoubleResponse_To_pb_ComputeDoubleResponse(in *inte
 
 ### How to add a new API group
 
-Simply create a new `api/<api_group_name>/<version>/api.proto` file, defining your new service; then generate the Go protobuf code, and run the CSI-proxy generator to generate all the boilerplate code.
+Simply create a new `client/api/<api_group_name>/<version>/api.proto` file, defining your new service; then generate the Go protobuf code, and run the CSI-proxy generator to generate all the boilerplate code.
 
 FIXME: add more details on which commands to run, and which files to edit when done generating.
```

Comment:
Open an issue to track this work. Also, it would be nice if it was all contained in a script

Reply:
Issue => https://github.com/kubernetes-csi/csi-proxy/issues/8
And that's what I'm currently working on anyway :)